SOOP/SOOP_CO2/incoming_handler.sh

```diff
@@ -26,11 +26,15 @@ is_future_reef_map_file() {
 main() {
     local file=$1; shift
     log_info "Handling SOOP CO2 zip file '$file'"
+    local recipient=`get_uploader_email $file`
 
     local tmp_dir=`mktemp -d`
     chmod a+rx $tmp_dir
     local tmp_zip_manifest=`mktemp`
 
+    echo "" | notify_by_email $BACKUP_RECIPIENT "Processing new underway CO2 zip file '$file'"
+    echo "" | notify_by_email $recipient "Processing new underway CO2 zip file '$file'"
+
```

Comment:
It would make sense to group these lines together with the `log_info` line above. The lines currently between them could come after these.